locale/compile-mo.sh

```diff
@@ -3,15 +3,15 @@
 # compile-mo.sh locale-dir/
 
 # Make this script fail if any command exits wit exit code != 0
-set -e
+set -ue
 
 function process_po_file() {
     pofile=$1
     dir=$(dirname "$pofile")
     lang=$(echo "$pofile" | cut -d "/" -f2)
     stem=$(basename "$pofile" .po)
     touch "${dir}/${stem}.mo"
-    dennis-cmd lint --errorsonly "$pofile" && msgfmt -o "${dir}/${stem}.mo" "$pofile"
+    dennis-cmd lint --quiet --errorsonly "$pofile" && msgfmt -o "${dir}/${stem}.mo" "$pofile"
```

Comment:
dennis's `quiet` mode is annoying - it doesn't print errors, the only thing we get is the non-zero return value. So if there is an error in a `.po` file the build would fail without more info - we wouldn't see what the error is or even which file it's coming from.

We maintain `dennis` and could add a new command-line flag or even release a new major version breaking backwards-compat on that...

Reply:
Or maybe you could do something like:

```suggestion
    dennis-cmd lint --quiet --errorsonly "$pofile" && msgfmt -o "${dir}/${stem}.mo" "$pofile" || dennis-cmd lint --errorsonly "$pofile"
```

? (Needs testing)

Reply:
Happy to just remove the --quiet. Just felt it was not outputting useful information just lines and lines telling us it is running.